runtime/src/main/java/org/corfudb/runtime/view/ObjectBuilder.java

```diff
@@ -116,6 +117,20 @@ public T open() {
                         getCorfuSMRProxy())
                         .getSerializer();
 
+                // FIXME: temporary hack until we have a registry
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)